src/main/groovy/com/bmuschko/gradle/vagrant/tasks/Vagrant.groovy

```diff
@@ -46,6 +51,7 @@ class Vagrant extends DefaultTask {
     @Input
     Map<String, String> environmentVariables = [:]
 
+    @Internal
```

Comment:
I think this field should be just `private final`. It's an implementation detail that should be exposed to the end user.

Reply:
The field was used in some of the tests which made it difficult to make it `private final`. I tried to have a second constructor so I can inject the mock in the tests into the tasks using the `TaskContainer.create` API but was unable to get that working. So in the end I annotated the field with `@PackageScope`. Hope that's also okay.

Reply:
I probably would have added a setter method with package private accessibility. I believe the annotation pretty much does the same.